src/pages/admin/alerts/index.tsx

```diff
@@ -15,7 +16,7 @@ const AdminMaintenanceAlertsOverviewEnglish: FC<DefaultSeoInfo> = ({ url }) => {
 export async function getStaticProps(
 	context: GetServerSidePropsContext
 ): Promise<GetServerSidePropsResult<DefaultSeoInfo>> {
-	return getDefaultStaticProps(context);
+	return getDefaultStaticProps(context, undefined, ROUTES_BY_LOCALE.en.adminAlerts);
```

Comment:
Consider swapping the order / switching to an `options` object so we can reduce these to e.g.
`getDefaultStaticProps(context, ROUTES_BY_LOCALE.en.adminAlerts)`

Reply:
fixed in commit on top of PR chain: https://github.com/viaacode/hetarchief-client/commit/f8ae3bf60cab6df8cf017cd5a24814c9568b5841